DesignRules.md

```diff
@@ -104,16 +104,11 @@ A resource describing a single thing is called a [=singular resource=]. Resource
       </dd>
       <dt>Implications</dt>
       <dd>
-         This rule is included in the automatic tests on <a href="https://developer.overheid.nl/">developer.overheid.nl</a>. The source code of the technical test can be found <a href="https://gitlab.com/commonground/don/adr-validator/-/blob/v0.5.0/pkg/rulesets/core/rules.go#L203">here</a>. The specific tests are published in the [[ADR-Validator]] repository.
-      </dd>      
+         This rule is included in the automatic tests on <a href="https://developer.overheid.nl/">developer.overheid.nl</a>. The specific tests are published in the [[ADR-Validator]] repository.
+      </dd>
       <dt>How to test</dt>
       <dd>
-         <ul>
-            <li> Step 1: The API MUST meet the prerequisets to be tested. These include that an OAS file is publicly available, parsable, all $refs are resolvable and paths are defined.</li>
-            <li> Step 2: Check if paths are present in the OpenAPI Specification.</li>
-            <li> Step 3: Loop all paths and check if it ends with a forward slash (<code>/</code>).</li>
-            <li> Step 4: Check all paths with a get request and without parameters. They SHOULD resolve in HTTP 404.</li>
-         </ul>
+         Loop all resource paths in the OpenAPI Specification and check that no resources paths end with a forward slash (<code>/</code>).
```

Comment:
Ik zou als term consequent "OpenAPI document" gebruiken, om verwarring te voorkomen met de standaard: de "OpenAPI Specification".

Reply:
Wellicht is "OpenAPI description" (OAD) ook een optie?

Reply:
OpenAPI document is de term die de standaard zelf hanteert, zie https://spec.openapis.org/oas/latest.html#openapi-document. Het lijkt me goed om dat over te nemen.

Reply:
Ah, ik zie nu dat OpenAPI Decription daar ook wordt genoemd, zie https://spec.openapis.org/oas/latest.html#openapi-description. Dat gaat dus om de inhoud van het document (los van de encoding). Dus bij nader inzien wellicht een beter term, goeie suggestie! 👍